### DISCUSSION TITLE
OpenAI integration

### DISCUSSION THREAD

**Discussion by gianlucamazza** [outline/outline#6472] [Ideas]
Hello everybody,

It would be great if  we can have openai assistant like inside outline.
I was thinking something like formatting or improving text, drawing charts and similar.
any similar feature planned?

great project. thank you!

**Comment by SaadBazaz** [comment]
Plane has a similar feature. I think this can be taken to another level by generalising the AI provider. (OpenAI, custom LLM, etc)

**Comment by klinshy** [comment]
Would be great to integrate custom LLMs
